examples/swapi/src/PersonById.re

```diff
@@ -0,0 +1,48 @@
+let ste = ReasonReact.string;
+
+type action =
+  | ChangeId(string);
+
+type state = {id: string};
+
+let handleClick = event =>
+  if (! ReactEventRe.Mouse.defaultPrevented(event)) {
+    ReactEventRe.Mouse.preventDefault(event);
+  };
+
+let reducer = (action, state) =>
+  switch (action) {
+  | ChangeId(id) => ReasonReact.Update({...state, id})
+  };
+
+let initialState = () => {id: ""};
+
+let onIdInputChange = ({ReasonReact.send}, e) =>
```

Comment:
I never seen doing this `{ReasonReact.send}`. What is the reason for that?

Reply:
In input:
```
 <input
          autoFocus=true
          placeholder="Get this id"
          _type="text"
          value=self.state.id
          onChange=(onIdInputChange(self))
          required=true
        />
```
i send the entire `self` object and its methods to a defined helper. Then used destructuring to pull off the `self.send` method and call it when the value change event is triggered.

Reply:
I mean, I get the destructuring but why no `({ send }, e) =>`?

I just tried and you get a

<img width="744" alt="image" src="https://user-images.githubusercontent.com/1110551/40478180-abfa2d36-5f48-11e8-920d-a551e3551d3c.png">

I guess it's because of this: https://reasonml.github.io/reason-react/docs/en/record-field-send-handle-not-found.html

Hence doing `ReasonReact.send`.

Something new I learned, thanks! 😄